### DISCUSSION TITLE
How can I open the modal QR code of WalletConnect

### DISCUSSION THREAD

**Discussion by d9dtanh** [rainbow-me/rainbowkit#612] [Q&A] [answered]
I follow the instructions in the doc of Rainbow Kit https://www.rainbowkit.com/docs/installation

But when I try to open modal qr-code WalletConnect, it doesn't working for me and throw an error like picture below

![image](https://user-images.githubusercontent.com/107018336/180735314-630d6044-1a40-48cc-aeb0-af3e3ffd70ec.png)

I try many way but it's not okay

I using
 - react version 18
 - rainbow kit version newest

could anyone help me and thanks for you support

**Comment by saihaj** [comment]
Can share a reproduction for this?

**Comment by markdalgleish** [comment, marked as answer]
You'll need to provide some polyfills since not all build tools provide them out of the box: https://www.rainbowkit.com/docs/installation#additional-build-tooling-setup.

Let us know if anything is missing!

**Comment by ameerhamza1214** [comment]
if you are using raibowkit and you want to customize the theme and overrides the classes in index.css or global.css it hide the qr of walletConnect 
try removing all custom css classes 
it works for me i was using rainbowkit in react js for multi wallet connection
